### PR TITLE
Use keyboard buttons to move player

### PR DESCRIPTION
**Pull request by zolter**
arrow + keypad

### CODE REVIEW

File: src/Game/Scenes/GameSceneInputHandler.cs [hunk 1]
```diff
@@ -52,11 +54,13 @@ partial class GameScene
         private Entity _queuedObject;
         private bool _rightMousePressed, _continueRunning;
 
-
         public bool IsMouseOverUI => Engine.UI.IsMouseOverAControl && !(Engine.UI.MouseOverControl is WorldViewport);
 
 	    private bool _isShiftDown;
 
+        [DllImport("user32.dll", CharSet = CharSet.Auto, ExactSpelling = true, CallingConvention = CallingConvention.Winapi)]
```
Comment (andreakarasho) [inline src/Game/Scenes/GameSceneInputHandler.cs]: it's not crossplatform!

**Comment by andreakarasho** [comment]
instead use the class GameSceneINputHandler
and inside the KeyDown event handle the arrow buttons to move char

File: src/Game/Scenes/GameSceneInputHandler.cs (hunk 1 above)
Comment (zolter) [reply src/Game/Scenes/GameSceneInputHandler.cs]: [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
        [return: MarshalAs(UnmanagedType.Bool)]
        private static extern bool SetDllDirectory(string lpPathName);

not crossplatform too ?

**Comment by zolter** [comment]
There were problems with this, now I will try again.

File: src/Game/Scenes/GameSceneInputHandler.cs (hunk 1 above)
Comment (andreakarasho) [reply src/Game/Scenes/GameSceneInputHandler.cs]: No, but when you use mono, we have the file .config that uses dllmap

**Comment by zolter** [comment]
> and inside the KeyDown event handle the arrow buttons to move char

No problem to handle the arrows. 
There is a problem with cross-platform getting numlock state. 
I'll try :)

**Comment by andreakarasho** [comment]
SDLK_NUMLOCKCLEAR

**Comment by zolter** [comment]
SDLK_NUMLOCKCLEAR - this is not a button state. 
Movement should occur only when the "Numlock" button is off

**Comment by andreakarasho** [comment]
(keymod & KMOD_NUM) == (KMOD_NUM)

**Comment by zolter** [comment]
ofc you right
strange why I forgot about it :/

**Comment by andreakarasho** [comment]
Is it ready?

**Comment by zolter** [comment]
Yep, it works correctly, as you wrote above. 
In the evening I will try to make the code smaller and remove 2 switch.

**Comment by zolter** [comment]
Is done

**Pull request merged**